NEWS.md

```diff
@@ -5,6 +5,15 @@ Python 3.6 support dropped
 
   afew stopped supporting the older python version 3.6.
 
+Handle DMARC report with empty spf or dkim XML nodes
+
+DMARC Filter: allow to define subject regexp
```

Comment:
https://github.com/afewmail/afew/pull/332 mentions in the PR description: 

Some DMARC report mail subject have a prefix before "Dmarc Report" in
the subject and are not checked by the plugin. This commit allows the
user to define the suject regexp in the config file.

Reply:
Ok